example/kubecon-2019-china/spark-sample/spark-pi.yaml

```diff
@@ -35,6 +35,8 @@ spec:
       hostPath:
         path: "/tmp"
         type: Directory
+  nodeSelector:
+    kubernetes.io/arch: amd64
```

Comment:
I don't think we need nodeSelector in this YAML.

Reply:
oh, yes, maybe we can add a new yaml for spark on amd64